tests/test_utils/test_transactiontestcase.py

```diff
@@ -26,3 +45,22 @@ def test(self, call_command):
             reset_sequences=False, inhibit_post_migrate=True,
             database='default', verbosity=0,
         )
+
+
+class TestDataConsistencyOnTearDown(TestSerializedContentMockMixin, TransactionTestCase):
+    """
+    TransactionTestCase._fixture_teardown() is flushing the whole database.
+    If an other test after this one needs some data coming from initial data migrations,
+    there data won't be found.
+    One solution is to load again initial data migrations after the flush called on _fixture_teardown().
+    """
+    available_apps = ['test_utils']
+    serialized_rollback = True
+    initial_data_migration = '[{"model": "test_utils.car", "pk": 1, "fields": {"name": "K 2000"}}]'
+
+    def _post_teardown(self):
+        super(TestDataConsistencyOnTearDown, self)._post_teardown()
+        self.assertTrue(Car.objects.exists())
```

Comment:
What I meant is, to leave the database in a clean state for other tests, we need:
```
self.assertTrue(Car.objects.exists())
Car.objects.delete()
```